css-canvas-medical/index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Challenge - Canvas Medical</title>
+    <link rel="stylesheet" href="fonts.css">
+    <link rel="stylesheet" href="reset.css">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class="container">
+
+      <div class="text-align-center">
+        <header class="row justify-between align-center gray-text page-header">
+            <div class="column-three-twentieths pointer">
+              <img src="images/logo-blue.svg">
+            </div>
+          <div class="column-full">
+            <nav class="justify-evenly">
+              <a href="#product">Product</a>
+              <a href="#implementation">Implementation</a>
+              <a href="#about-us">About Us</a>
+              <a href="#blog">Blog</a>
+              <button class="contact-us">Contact Us</button>
```

Comment:
This is still used for navigation, it should be an `a` element, you can just style it to look like a button